test/autotest.py

```diff
@@ -292,7 +360,8 @@ def runCmd(cmd):
 ckptDir="dmtcp-autotest-%d" % randint(100000000,999999999)
 os.mkdir(ckptDir);
 os.environ['DMTCP_COORD_HOST'] = "localhost"
-os.environ['DMTCP_COORD_PORT'] = str(randint(2000,10000))
+if not os.getenv('DMTCP_COORD_PORT'):
```

Comment:
We need to revert this change. I often have a coordinator already running at `DMTCP_COORD_PORT` (which is defined in the environment). With this change, the tests will try to connect to the existing coordinator.

Instead, we should make it something like:

    if not os.getenv('DMTCP_NUM_RUN'):
      os.environ['DMTCP_COORD_PORT'] = str(randint(2000,10000))